src/main/java/org/apache/commons/text/names/Name.java

```diff
@@ -26,14 +26,16 @@
 public final class Name {
 
     private final String leadingInitial;
+    private final String salutation;
     private final String firstName;
     private final String nickName;
     private final String middleName;
     private final String lastName;
     private final String suffix;
 
-    Name(String leadingInitial, String firstName, String nickName, String middleName, String lastName, String suffix) {
+    public Name(String leadingInitial, String salutation, String firstName, String nickName, String middleName, String lastName, String suffix) {
```

Comment:
Ditto the above, no need to make it public for the salutations; so let's keep the pull request simple and implement just the salutations for now ;-)